builtin/logical/pki/path_roles.go

```diff
@@ -344,8 +375,11 @@ func (b *backend) pathRoleCreate(
 		KeyUsage:            data.Get("key_usage").(string),
 		OU:                  data.Get("ou").(string),
 		Organization:        data.Get("organization").(string),
+		GenerateLease:       new(bool),
```

Comment:
Yeah, it does need the space allocated. You could do it all below but easy to just do it here.